### ISSUE TITLE
Bug getting queues with regex with RabbitMQ Scaler

### ISSUE THREAD

**Issue by JorTurFer** [kedacore/keda#2054]
### Report

I have to apologize because the RabbitMQ Scaler is not working with regex :( 

Idk why I totally forgot 2 important things which should be part of the query string, the page (with `pagination=false` is still needed the parameter `page=1` ) and the regex in url format. 
I'm really sorry about it and I will fix it ASAP (I start with this now) and I will alse create idk how an integration test to ensure that it works

And sorry again :(

### Expected Behavior

Use regex and it works

### Actual Behavior

Now all the queues are returned

### Steps to Reproduce the Problem

1.
2.
3.

### Logs from KEDA operator

```
example
```

### KEDA Version

2.4.0

### Scaler Details

RabbitMQ

**Comment by zroubalik** [comment]
OK, no worrries.

wrt testing, the best thing would be to extend current RabbitMQ e2e test and add regex scenario there:  https://github.com/kedacore/keda/tree/main/tests

**Comment by JorTurFer** [comment]
The problem is that I did the test, but it is wrong... I will fix it also :)

**Comment by JorTurFer** [comment]
Hi,
next release will be in the first half of November, AFAIR. Probably you are getting errors with helm and chart because and argument has changed after the kubebuilder updates.
Before the release, you need to [update this line](https://github.com/kedacore/charts/blob/master/keda/templates/12-keda-deployment.yaml#L52) after release the chart, or maybe you can use the chart from your local

**Comment by crisp2u** [comment]
@JorTurFer yeah, that's the issue, i did not want to go into details of that. I've also tried to escape metricName and queueName and add `page=1` outside but still no luck
